### PR TITLE
feat: Add github star count to Getting Started

### LINKED ISSUE #1619: Add Github star count to docs page
Add a big star count [Github button](https://buttons.github.io/) to https://wave.h2o.ai/ (probably under Get started button).

### PR DESCRIPTION
**Pull request by JacksonIsaac**
# Github issue
Closes #1619 

## Output
image

#### Mobile version
image

### CODE REVIEW

**Review by mturoci — CHANGES_REQUESTED** [review]
Thanks @JacksonIsaac! The implementation looks fine, but is not in the good place. Please read the issue deacription carefully and move the button to the landing page.

**Comment by JacksonIsaac** [comment]
I misread it indeed! Should be fixed now :) Although I don't really like the alignment though. Any tips perhaps ? I did try some iframe align parameters, but nothing really changed anything

**Comment by mturoci** [comment]
As for the alignment, I would suggest looking at CSS alignment options.

**Comment by JacksonIsaac** [comment]
I like this better :) Updated also the sample output in the description

File: website/src/pages/index.js [hunk 1]
```diff
@@ -106,6 +106,8 @@ function Home() {
               Get Started
             </Link>
           </div>
+          <p className="hero__subtitle"></p>
```
Comment (mturoci) [inline website/src/pages/index.js:109]: What is this for?

File: website/src/pages/index.js [hunk 2]
```diff
@@ -106,6 +106,8 @@ function Home() {
...
+          <iframe src="https://ghbtns.com/github-btn.html?user=h2oai&repo=wave&type=star&count=true&size=large" style={{marginLeft:2+'em'}} align="middle" frameborder="0" scrolling="0" width="170" height="30" title="GitHub"></iframe>
```
Comment (mturoci) [inline website/src/pages/index.js:110]: The line is way too long, can you please put every attr into a newline?

**Review by mturoci — CHANGES_REQUESTED** [review]
Thanks @JacksonIsaac! A few minor comments and we are good to go!

File: website/src/pages/index.js (hunk 1 above)
Comment (JacksonIsaac) [reply website/src/pages/index.js:109]: This is to add spacing between Get Started button and GH star button

File: website/src/pages/index.js (hunk 2 above)
Comment (JacksonIsaac) [reply website/src/pages/index.js:110]: Will do :) The marginLeft is to add spacing on left and make it center aligned as compared to Get Started button. Added a screenshot with mobile version to description

**Review by mturoci — APPROVED** [review]
Thanks @JacksonIsaac!

**Pull request merged**